project_generator/project.py

```diff
@@ -172,40 +172,39 @@ def __init__(self, name, project_files, workspace):
             with open(project_file, 'rt') as f:
                 project_file_data = yaml.load(f)
 
-            if 'common' in project_file_data:
-                group_name = 'default'
-                if 'group_name' in project_file_data['common']:
-                    group_name = project_file_data['common']['group_name'][0]
+            group_name = 'default'
```

Comment:
Can you revert this ? This change was done so yaml file without common: can be supported. For example when there's only tool specific in the yaml defined.

```
tool_specific:
  iar:
    -something here
```